src/components/pages/courses/course-page-layout.tsx

```diff
@@ -122,6 +149,36 @@ const CoursePageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
                   )}
                 </UserRating>
               </div>
+
+              <div className="flex items-center md:justify-start justify-center mt-4 space-x-2">
```

Comment:
rss and download links!

Reply:
![image](https://user-images.githubusercontent.com/86834/103467041-b01d1700-4cff-11eb-9c86-f1ef352d512f.png)